### PR TITLE
[8.0][ADD] module_repository

### PR DESCRIPTION
**Pull request by legalsylvain**
Hi all,

I Propose a module named ``module_repository``. This module is usefull, only if you deploy Odoo via github : buildout, voodoo, manual git clone, etc...

(If you use installation by pip, or the nightly build, this module is useless)

It add a new model ``ir.module.repository``, by analyzing ``addons_path`` value

ir_module_repository_kanban

ir_module_repository_form

### CODE REVIEW

File: module_repository/models/ir_module_repository.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            if repository.installed_module_qty == 0:
+                repository.state = 'unused'
+            else:
+                repository.state = 'used'
+
+    @api.model
+    def _create_repositories_from_addons_path(self):
+        _logger.info("Update Repositories List from addons_path ...")
+        addons_paths = config['addons_path']
+        for addons_path in [x.strip() for x in addons_paths.split(',')]:
```
Comment (bealdav) [inline module_repository/models/ir_module_repository.py]: nitpicking: probably addons_paths var is useless because only used once.

Comment (legalsylvain) [reply module_repository/models/ir_module_repository.py]: Thanks. done.

**Comment by legalsylvain** [comment]
Note : Travis error is unrelated. Branch 8.0 is broken on master. (also 9.0 by the way...)

image

**Review by sergiocorato — APPROVED** [review]

**Comment by sergiocorato** [comment]
> Note : Travis error is unrelated. Branch 8.0 is broken on master. (also 9.0 by the way...)
> 
> image

@legalsylvain re-launched and passed all checks

**Comment by legalsylvain** [comment]
Thanks for your review !
/ocabot merge

**Pull request closed without merging**